nix/ovpn.nix

```diff
@@ -0,0 +1,43 @@
+{ config, pkgs, ... }:
+{
+  boot = {
+    kernel.sysctl = {
+      "net.ipv4.ip_forward" = 1;
+    };
+  };
+  networking = {
+    firewall = {
+      enable = true;
+      extraCommands = ''
+        iptables -t nat -A POSTROUTING -s 10.8.0.0/24 -o eno1 -j MASQUERADE
+      '';
+      extraStopCommands = ''
+        iptables -t nat -D POSTROUTING -s 10.8.0.0/24 -o eno1 -j MASQUERADE
+      '';
+      allowedUDPPorts = [ 1195 ]; # openvpn
+      trustedInterfaces = [ "tun1" ];
+    };
+  };
+  services = {
+    openvpn = {
+      servers = {
+        for-phone = {
+          config = ''
+            dev tun1
+            server 10.8.0.0 255.255.255.0
+            port 1195
+            comp-lzo
+            ca /root/easy-rsa/easyrsa3/pki/ca.crt
+            cert /root/easy-rsa/easyrsa3/pki/issued/no-pass.crt
```

Comment:
hmm...

Sorry, so whats the security situation here? We have a private key on the clients, and then install the public keys server side?

That SG to me, but i wonder why the default is to require pass. I might leave it on just cause I'm nervous.